app/views/students/_index.html

```diff
@@ -0,0 +1,107 @@
+<div ng-controller='StudentsIndexCtrl as controller'>
+	<div id="wrapper">
+		<div id="page-wrapper">
+			<div class="container-fluid">
```

Comment:
There is already a container class applied in the main's web index and bootstrap containers should not be nested. Please check this.